homework-1.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title></title>
+
+    <!-- Bootstrap -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+  </head>
+  <body>
+    <h2>Add Item</h2>
+    <h3> Item Name: </h3>
+    <input id="name" placeholder="Item Name" width="400px"/><br>
+    <h3> Price: </h3>
+    <input id="price" placeholder="Price" type="number"/><br>
+    <p><h3>Tags:</h3>
+    Travel <input type="checkbox" class="tags" name="travel"><br>
+    Home Improvement <input type="checkbox" class="tags" name="Home Improvement"><br>
+    Food and Household <input type="checkbox" class="tags" name="Food and Household"><br>
+    Luxury <input type="checkbox" class="tags" name="Luxury"><br>
+    Electronics <input type="checkbox" class="tags" name="Electronics"><BR>
+    Gardening <input type="checkbox" class="tags" name="Gardening"><br>
+    Fashion <input type="checkbox" class="tags" name="Fashion"><br>
+    </p>
+    <p> <h3>Stores:</h3>
+      Walmart <input type="checkbox" class="stores" name="Walmart"><br>
+      Target <input type="checkbox" class="stores" name="Target"><br>
+      Whole Foods <input type="checkbox" class="stores" name="Whole Foods"><br>
+      Home Depot <input type="checkbox" class="stores" name="Home Depot"><br>
+      Amazon <input type="checkbox" class="stores" name="Amazon"><br>
+      Costco <input type="checkbox" class="stores" name="Costco"><br>
+      Walgreens <input type="checkbox" class="stores" name="Walgreens"><br>
+    </p>
+    <button id="add">Sumbit Item</button>
+    <h2>Or look at the inventories of some stores</h2>
+      Walmart <input type="checkbox" class="storesToLookAt" name="Walmart"><br>
+      Target <input type="checkbox" class="storesToLookAt" name="Target"><br>
+      Whole Foods <input type="checkbox" class="storesToLookAt" name="Whole Foods"><br>
+      Home Depot <input type="checkbox" class="storesToLookAt" name="Home Depot"><br>
+      Amazon <input type="checkbox" class="storesToLookAt" name="Amazon"><br>
+      Costco <input type="checkbox" class="storesToLookAt" name="Costco"><br>
+      Walgreens <input type="checkbox" class="storesToLookAt" name="Walgreens"><br>
+    <button id="look">Look at  these stores</button>
+    <p id="storeInventories"></p>
+    <script>
+    class Store {
+      constructor(name) {
+        this.name=name;
```

Comment:
Spaces :)

```js
this.name = name;
```

https://github.com/airbnb/javascript#whitespace

There are more lines with this issue than the ones I mentioned.